next/components/search/search-multicheckbox-facet.tsx

```diff
@@ -21,10 +21,10 @@ export function MultiCheckboxFacet({
 }: FacetViewProps) {
   const { t } = useTranslation();
   return (
-    <fieldset className={clsx(className, "mb-4")}>
-      <legend className="mb-5 text-heading-xs font-bold text-steelgray">
+    <section className={clsx(className, "mb-4")}>
```

Comment:
I think we can actually remove the `className` prop from this entirely - this component is only used in one place and the className isn't used

Reply:
Also, I think a `fieldset` is better than a `section` here? Fieldset is used to group related form elements together, which is exactly what this is. I think `section` elements are for bigger building blocks of a page, not for this case where it's just a few form elements.